db/seeds.d/50-bootdisk_templates.rb

```diff
@@ -1,8 +1,8 @@
-kind = TemplateKind.find_or_create_by_name('Bootdisk')
+kind = TemplateKind.find_or_create_by(:name => 'Bootdisk')
 
 ProvisioningTemplate.without_auditing do
   content = File.read(File.join(ForemanBootdisk::Engine.root, 'app', 'views', 'foreman_bootdisk', 'host.erb'))
-  tmpl = ProvisioningTemplate.find_or_create_by_name(
+  tmpl = ProvisioningTemplate.find_or_create_by(
```

Comment:
This isn't equivalent, it's find or creating by all of those attributes and not solely by name.